deploy/helm/sumologic/templates/setup/secret.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "sumologic.metadata.name.setup.secret" . }}
+  annotations:
+{{ include "sumologic.annotations.app.setup.helmsh" "3" | indent 4 }}
+  labels:
+    app: {{ template "sumologic.labels.app.setup.secret" . }}
+    {{- include "sumologic.labels.common" . | nindent 4 }}
+data:
+  sumologic_accessid: {{ required "A valid .Values.sumologic.accessId entry required!" .Values.sumologic.accessId | b64enc }}
+  sumologic_accesskey: {{ required "A valid .Values.sumologic.accessKey entry required!" .Values.sumologic.accessKey | b64enc }}
```

Comment:
I'm thinking about making it optional, so customer can provide it by hand before installation

Reply:
Hm, there's already https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/7ff4d7aa7b6025f613e7dbe1d7aaa043ef27c9d6/deploy/helm/sumologic/templates/setup/job.yaml#L64, I guess I could only create the secret if it's not set, and document appropriately.

Reply:
:+1: 

It will avoid raising `A valid .Values.sumologic.accessId entry required!` if secret is set by hand